code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -174,6 +174,17 @@
 	new /obj/item/weapon/gun/projectile/pistol(src)
 	new /obj/item/weapon/silencer(src)
 
+/obj/item/weapon/storage/box/syndie_kit/g9mm/plus
+	name = "smooth operator+"
+	desc = "9mm with silencer kit, and extra ammo."
+
+/obj/item/weapon/storage/box/syndie_kit/g9mm/plus/fill()
+	..()
```

Comment:
I think that is going to end up with two pistols and two silencers as you are calling the parent that already fills the box with a pistol and a silencer

Reply:
Might be worth just changing the normal kit to include two spare mags

Reply:
Fixed